bubblegum/program/tests/simple.rs

```diff
@@ -4,15 +4,18 @@ use solana_program_test::tokio;
 use solana_sdk::signature::{Keypair, Signer};
 
 use utils::{
-    clone_keypair,
     context::{BubblegumTestContext, DEFAULT_LAMPORTS_FUND_AMOUNT},
-    LeafArgs, Result, Tree,
+    tree::Tree,
+    LeafArgs, Result,
 };
 
 // Test for multiple combinations?
-const MAX_DEPTH: usize = 20;
+const MAX_DEPTH: usize = 14;
```

Comment:
I lowered it because the default construction used in this test code for the proof `MerkleTree` here populates the entire tree, and for `1 << 20` it took quite a while.